contracts/AnchoredView/AnchoredView.sol

```diff
@@ -190,50 +178,50 @@ contract AnchoredView is SymbolConfiguration {
      * @param cToken The cToken address for price retrieval
      * @return The price for the given cToken address
      */
-    function getUnderlyingPrice(address cToken) external view returns (uint256) {
+    function getUnderlyingPrice(address cToken) public returns (uint) {
```

Comment:
what happens, if anchor is stale here?

Reply:
if anchor is stale here, kind of doomed. We really can't let anchor go down until Reporters cover all markets